python-script/augment_GMM.py

```diff
@@ -49,13 +59,14 @@ def plot_mahalanobis_distance_distribution(X, Q, dataset):
         # X = read_fbin(data_path)
         D = X.shape[1]
         print(X.shape)
-        # X = L2_norm_dataset(X)
+        X = L2_norm_dataset(X)
```

Comment:
The GMM training functions in the original library were all commented out. When I was using it, I uncommented them all in bulk because I wasn't sure which lines were necessary and which weren't, so this line also got uncommented.